<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Macooks-MacBook-Pro.local" failures="1" tests="6" name="smokeTests.ReqResCrudTest" time="0.635" errors="0" timestamp="2025-08-10T00:45:04 EEST" skipped="1">
  <testcase classname="smokeTests.ReqResCrudTest" name="deleteUserTest" time="0.075">
    <failure message="
Expecting actual:
  401
to be in:
  [204, 200]
" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
Expecting actual:
  401
to be in:
  [204, 200]

at smokeTests.ReqResCrudTest.deleteUserTest(ReqResCrudTest.java:52)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- deleteUserTest -->
  <testcase classname="smokeTests.ReqResCrudTest" name="readUserTest" time="0.053"/>
  <testcase classname="smokeTests.ReqResCrudTest" name="partialUpdateUserTest" time="0.097"/>
  <testcase classname="smokeTests.ReqResCrudTest" name="deleteUserTest" time="0.075">
    <skipped/>
  </testcase> <!-- deleteUserTest -->
  <testcase classname="smokeTests.ReqResCrudTest" name="createUserTest" time="0.234"/>
  <testcase classname="smokeTests.ReqResCrudTest" name="updateUserTest" time="0.101"/>
</testsuite> <!-- smokeTests.ReqResCrudTest -->
